### PR TITLE
feat(earn): fetch apy from contract

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

Add a fetch function that gets APY from aave contract

### Test plan

Unit tests, manual

### Related issues

- Fixes ACT-1174

### Backwards compatibility

Yes

### Network scalability

N/A

### CODE REVIEW

File: src/earn/EarnCta.tsx [hunk 1]
```diff
@@ -28,17 +46,21 @@ export default function EarnCta() {
...
-                {/* TODO(ACT-1174): use the right amounts here */}
-                <TokenDisplay
-                  amount={10}
-                  tokenId={networkConfig.arbUsdcTokenId}
-                  showLocalAmount={false}
-                />
-                <TokenDisplay amount={1} tokenId={networkConfig.arbUsdcTokenId} />
-              </Trans>
-            </Text>
+            {asyncPoolInfo.result && (
```
Comment (satish-ravi) [inline src/earn/EarnCta.tsx:49]: Still need to handle loading / error states. Waiting on design

File: src/earn/EarnCta.tsx [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
 import EarnAave from 'src/icons/EarnAave'
 import Colors from 'src/styles/colors'
 import { typeScale } from 'src/styles/fonts'
 import { Spacing } from 'src/styles/styles'
+import { useTokenInfo } from 'src/tokens/hooks'
+import Logger from 'src/utils/Logger'
 import networkConfig from 'src/web3/networkConfig'
+import { Address } from 'viem'
+
+const USDC_AMOUNT = 10
```
Comment (satish-ravi) [inline src/earn/EarnCta.tsx:19]: Waiting on product for actual amount to display

File: src/earn/poolInfo.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import { ensureError } from 'src/utils/ensureError'
+import { publicClient } from 'src/viem'
+import networkConfig from 'src/web3/networkConfig'
+import { Address } from 'viem'
+
+const TAG = 'earn/poolInfo'
+
+const COMPOUND_PERIOD = 365 * 24 * 60 * 60 // 1 year in seconds
+
+export async function getAavePoolInfo(assetAddress: Address) {
```
Comment (MuckT) [inline src/earn/poolInfo.ts:14]: nit: naming `fetchAavePoolInfo` to match what we do with jumpstart.

File: src/web3/networkConfig.ts [hunk 4]
```diff
@@ -141,9 +142,12 @@ const CREAL_TOKEN_ID_MAINNET = `${NetworkId['celo-mainnet']}:0xe8537a3d056da4466
 const ETH_TOKEN_ID_STAGING = `${NetworkId['ethereum-sepolia']}:native`
 const ETH_TOKEN_ID_MAINNET = `${NetworkId['ethereum-mainnet']}:native`
 
-const ARB_USDC_TOKEN_ID_STAGING = `${NetworkId['arbitrum-sepolia']}:0xf3c3351d6bd0098eeb33ca8f830faf2a141ea2e1`
+const ARB_USDC_TOKEN_ID_STAGING = `${NetworkId['arbitrum-sepolia']}:0x75faf114eafb1bdbe2f0316df893fd58ce46aa4d`
 const ARB_USDC_TOKEN_ID_MAINNET = `${NetworkId['arbitrum-one']}:0xaf88d065e77c8cc2239327c5edb3a432268e5831`
 
+const ARB_AAVE_CONTRACT_ADDRESS_STAGING = '0xBfC91D59fdAA134A4ED45f7B584cAf96D7792Eff'
+const ARB_AAVE_CONTRACT_ADDRESS_MAINNET = '0x794a61358D6845594F94dc1DB02A252b5b4814aD'
```
Comment (jh2oman) [inline src/web3/networkConfig.ts:149]: Nit: change the name to specify that this is the Pool contract.

Comment (silasbw) [reply src/web3/networkConfig.ts:149]: can you add "v3" too?  e.g., `...AAVE_POOL_V3_CONTRACT...`

File: src/abis/AavePool.ts
Comment (MuckT) [inline src/abis/AavePool.ts:1]: Is there a place to verify the contents of this file for Aave?

Comment (silasbw) [reply src/abis/AavePool.ts:1]: nit: add the V3 to the filename

Comment (satish-ravi) [reply src/abis/AavePool.ts:1]: @MuckT added a link to arbiscan and docs for ABI
@silasbw renamed

**Review by MuckT — APPROVED** [review]

**Review by silasbw — APPROVED** [review]

**Pull request merged**